lrmd/lrmd.c

```diff
@@ -1654,6 +1673,36 @@ process_lrmd_rsc_exec(crm_client_t * client, uint32_t id, xmlNode * request)
 }
 
 static int
+process_lrmd_alert_exec(crm_client_t * client, uint32_t id, xmlNode * request)
+{
+    lrmd_rsc_t *alert = NULL;
+    lrmd_cmd_t *cmd = NULL;
+    xmlNode *alert_xml = get_xpath_object("//" F_LRMD_RSC, request, LOG_ERR);
+    const char *alert_id = crm_element_value(alert_xml, F_LRMD_RSC_ID);
+    int call_id;
+
+    if (!alert_id) {
+        return -EINVAL;
+    }
+
+    if (!(alert = g_hash_table_lookup(rsc_list, alert_id))) {
```

Comment:
I prefer to put the assignment on a separate line from the test. It's easier to read, and makes it clear it's not a forgotten "==".